docs/add_codechecker_to_a_project.md

```diff
@@ -0,0 +1,42 @@
+We can analyze code by using the CodeCompass with codechecker parameter.
+
+# :exclamation: Prerequisites
+* You must first parse your project first using CodeCompass parse command based on CodeCompass Parse Guide 
+*Download and install the CodeChecker tool https://github.com/Ericsson/codechecker
```

Comment:
We should mention that upstream CodeChecker also doesn't support this sort of operation... Maybe a link to the tagged version of 5.2 or something like that should be added here, instead.

Reply:
What sort of operation?

Reply:
https://github.com/Ericsson/codechecker/tree/v5.10
operation := using codechecker together with codecompass